README.md

```diff
@@ -79,10 +79,17 @@ Schedules a task to be run on a Worker thread.
 * `filename`: Optionally overrides the `filename` option passed to the
   constructor for this task. If no `filename` was specified to the constructor,
   this is mandatory.
+* `abortSignal`: An [`AbortSignal`][] instance. If passed, this can be used to
+  cancel a task, including already running ones.
+  (More generally, any `EventEmitter` or `EventTarget` that emits `'abort'`
+  events can be passed here.) Abortable tasks cannot share threads regardless
+  of the `concurrentTasksPerWorker` options.
 
 This returns a `Promise` for the return value of the (async) function call
 made to the function exported from `filename`. If the (async) function throws
 an error, the returned `Promise` will be rejected with that error.
+If the task is aborted, the returned `Promise` is rejected with an error
+as well.
 
```

Comment:
Can you please expand to what happens to the worker if the task is aborted while it was running?

Reply:
Done, but imo it’s an implementation detail that we should not focus on here.